app/controllers/simplified-account/settings/team-members/change-permission/change-permission.controller.js

```diff
@@ -0,0 +1,33 @@
+const { findByExternalId } = require('@services/user.service')
+const { response } = require('@utils/response')
+const formatSimplifiedAccountPathsFor = require('../../../../../utils/simplified-account/format/format-simplified-account-paths-for')
```

Comment:
You can use `require('@utils/...')` here as well

Reply:
done